### DISCUSSION TITLE
The autofill exe cannot detect XML in the same directory even though both files are in the same folder

### DISCUSSION THREAD

**Discussion by TrueBlue726** [chilli-axe/mpc-autofill#214] [General]
See the title.  I have downloaded the latest software for autofill.exe, and after I downloaded the XML from MPCFill.com, I put that file in the same folder as the executable and ran the exe.  After several questions (is that supposed to happen every time I run the exe?), the program would say no XML is detected within the same directory.  I am not sure what I did wrong.  Please give me a detailed instruction on how to run the program.  I have a 612 card order that I need to have it shipped to me ASAP, so any help is appreciated.  
![Folder](https://github.com/chilli-axe/mpc-autofill/assets/95557492/9332ed18-afdd-4890-a0b9-076f50f41714)
![NO XML FILES FOUND IN THIS DIRECTORY](https://github.com/chilli-axe/mpc-autofill/assets/95557492/9be5c364-b402-4726-8d7d-612616ac8e55)

**Comment by ndepaola** [comment]
howdy! it looks like these files are located in onedrive - could you try moving them to a local directory (e.g. your desktop) and try again? i haven't tried running the program in a onedrive directory before but it wouldn't be surprising if that's the problem here.

your description of how you're running the program is correct and it's expected that it asks you those questions at the start each time

**Comment by TrueBlue726** [comment]
Thank you very much.  That solved the issue.

**Comment by ndepaola** [comment]
glad to hear it! i'll prepare a bug report for this - what version of windows are you running?

**Comment by TrueBlue726** [comment]
Windows 11 Build 10.10.22631

**Comment by ndepaola** [comment]
cheers! closing this thread now as i'm tracking the issue in #215
